gemfiles/rails_4_2.gemfile

```diff
@@ -5,5 +5,6 @@ source "https://rubygems.org"
 gem "railties", "~> 4.2.1"
 gem "i18n", "< 1.5.1", platform: :mri_22
 gem "capybara", "< 3.16.0", platform: [:mri_23, :mri_22]
+gem "nokogiri", "< 1.9.1", platform: [:mri_23, :mri_22]
```

Comment:
Shoudn't this line also appear in the Appraisals file?

Reply:
"Possibly". It technically isn't needed, but this line is added to "force" travis' dodgy methodology of installing bad gems.

When running appraisals locally you'll always get the right version. I can add it to the Appraisals if you want standardisation, but to answer your question; "No" it isn't +needed+